django_fido/models.py

```diff
@@ -158,10 +158,13 @@ def metadata(self) -> Optional['AuthenticatorMetadata']:
             return metadata
         # Take the device certificate and try to validate against all certs in MDS
         device_cert = crypto.load_certificate(crypto.FILETYPE_ASN1, self.attestation.att_statement['x5c'][0])
-        try:
+        if metadata.detailed_metadata_entry != '':
             root_certs = json.loads(metadata.detailed_metadata_entry)['attestationRootCertificates']
```

Comment:
* Is the `attestationRootCertificates` key always present? It will throw KeyError if not.
* The `else` branch checks that `root_certs` is not empty. Don't we need the same check here as well?

Reply:
In MDS 2.0 the key is required, so there shouldn't be any need for that.